site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -0,0 +1,388 @@
+---
+title: 'Low-level API'
+linkTitle: 'Low-level API'
+weight: 3
+description: ''
+---
+
+## Overview
+
+Low-level API is useful if you need to work directly with REST API, but want
+to have data validation and syntax assistance from your code editor. The code
+on this layer is autogenerated.
+
+Code of this component is located in `cvat_sdk.api_client`.
+
+## Example
+
+Let's see how a task with local files can be created. We will use the basic auth
+to make things simpler.
+
+```python
+from time import sleep
+from cvat_sdk.api_client import Configuration, ApiClient, models, apis, exceptions
+
+configuration = Configuration(
+    host="http://localhost",
+    username='YOUR_USERNAME',
+    password='YOUR_PASSWORD',
+)
+
+# Enter a context with an instance of the API client
+with ApiClient(configuration) as api_client:
+    # Parameters can be passed as a plain dict with JSON-serialized data
+    # or as model objects (from cvat_sdk.api_client.models), including
+    # mixed variants.
+    #
+    # In case of dicts, keys must be the same as members of models.I<ModelName>
+    # interfaces and values must be convertible to the corresponding member
+    # value types (e.g. a date or string enum value can be parsed from a string).
+    #
+    # In case of model objects, data must be of the corresponding
+    # models.<ModelName> types.
+    #
+    # Let's use a dict here. It should look like models.ITaskWriteRequest
+    task_spec = {
+        'name': 'example task',
+        "labels": [{
+            "name": "car",
+            "color": "#ff00ff",
+            "attributes": [
+                {
+                    "name": "a",
+                    "mutable": True,
+                    "input_type": "number",
+                    "default_value": "5",
+                    "values": ["4", "5", "6"]
+                }
+            ]
+        }],
+    }
+
+    try:
+        # Apis can be accessed as ApiClient class members
+        # We use different models for input and output data. For input data,
+        # models are typically called like "*Request". Output data models have
+        # no suffix.
+        (task, response) = api_client.tasks_api.create(task_spec)
+    except exceptions.ApiException as e:
+        # We can catch the basic exception type, or a derived type
+        print("Exception when trying to create a task: %s\n" % e)
+
+    # Here we will use models instead of a dict
+    task_data = models.DataRequest(
+        image_quality=75,
+        start_frame=2,
```

Comment:
The parameters look strange for data that contains only 2 files.

Reply:
Removed